Documentation/security/threat-model.rst

```diff
@@ -0,0 +1,751 @@
+|                  |                                                   |
+|                  | -  In overlay mode, the attacker could inspect    |
+|                  |    communication between workloads, and spoof     |
+|                  |    traffic that appears to come from any          |
+|                  |    workload via spoofing identities               |
+|                  | -  In native routing mode, the attacker could     |
+|                  |    inspect communication between workloads. If    |
+|                  |    the attacker was able to control the           |
+|                  |    ipcache through another attack (for            |
+|                  |    example, by either being a successful          |
+|                  |    :ref:`API server attacker                      |
+|                  |    <kubernetes-api-server-attacker>`              |
+|                  |    or :ref:`key-value store attacker              | 
+|                  |    <kv-store-attacker>`                           |
+|                  |    they could also spoof traffic by spoofing      |
+|                  |    pod identities.                                |
+|                  |                                                   |
+|                  | Denial of service could occur depending on the    |
+|                  | behavior of the attacker.                         |
++------------------+---------------------------------------------------+
+| Observability    | TLS is required for all connectivity between      |
+| data             | Cilium components, as well as for exporting       |
+|                  | data to other destinations (such as a S3,         |
+|                  | Splunk, or Elastic), removing the scope for       |
+|                  | spoofing or tampering.                            |
+|                  | |br| |br|                                         |
+|                  | Without transparent encryption, the attacker      |
+|                  | could re-create the observability data as         |
+|                  | available on the network level.                   |
+|                  | |br| |br|                                         |
+|                  | Information leakage could occur via an attacker   |
+|                  | scraping Hubble Prometheus metrics. These metrics |
+|                  | are disabled by default, and depending on         |
+|                  | configuration can contain sensitive information   |
+|                  | on network flows.                                 |
+|                  | |br| |br|                                         |
+|                  | Denial of service could occur depending on the    |
+|                  | behavior of the attacker.                         |
++------------------+---------------------------------------------------+
+
+Recommended Controls
+^^^^^^^^^^^^^^^^^^^^
```

Comment:
The equivalent previous sections used bulletpoints for this section, it would be good to be consistent.